### DISCUSSION TITLE
If you had to recommend a single work-related book, what would it be and why?

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1400] [Chat about dev stuff]
As in the title, do you have any coding-related, or in general work or career-related books you would like to recommend?

**Update:** As of now our list contains:

- "Refactoring UI" by Adam Wathan
- "Atomic Design" by Brad Frost
- "Don't Make Me Think" by Steve Krug
- "Getting Things Done" by David Allen
- "A Mind for Numbers Up" by Oakley Barbara
- "Clean Code" by Robert C. Martin
- "JavaScript: The Good Parts" by Douglas Crockford

Keep it up!

**Comment by filipkappa** [comment]
For my part, I really enjoyed reading the book penned by the TailwindCSS creator himself called [Refactoring UI](https://www.refactoringui.com/) by @adamwathan & @sschoger.

The most important thing about that book is that it teaches you how to create great UI by using very down-to-earth examples and actionable tips, which I was able to use right away.

The main takeaways I still pay attention to, to this day are:

- I started using hsl instead of HEX or RGB color formats wherever I can, as it's much more intuitive
- I try to use space as a tool for sectioning & visually grouping content on a page

**What about you?**

**Comment by codewithmenow** [comment]
"Atomic Design" by @bradfrost

It's not so much about coding, more like "building design systems".

This book breaks down UI design into its tiniest building blocks and shows you how to assemble them into bigger, better components.

I would recommend it to anyone that's trying to build something that's both consistent and pretty.

**Reply by filipkappa** [reply]
Awesome, I'll check it out fo sure!

**Comment by cyggarro** [comment]
"Don't Make Me Think" by Steve Krug

**Reply by filipkappa** [reply]
Do you have any interesting takeaways from it?

**Comment by ascensus-mdb** [comment]
Not related to coding and I read it a dozen or so years ago, but this is the book that has had the biggest impact on my work ethic and approach to productivity.

An all-time classic, "Getting Things Done" by David Allen.

**Comment by bwsky-a** [comment]
It may not be a book related to coding, but it allows us to get acquainted with numbers and ways to learn more effectively.

"A Mind for Numbers Up" by Oakley Barbara

**Comment by DanWearr** [comment]
One of my favorite work-related books is "Clean Code" by Robert C. Martin. This book is a must-read for any developer who wants to write better code that is easier to maintain and understand. 

It has examples for writing code that is **readable**, **testable**, and **easy to refactor** - 3 key characteristics that are often missing, especially in newbie projects.

If you haven't read it yet, I highly recommend it!

**Comment by garyross4** [comment]
+1 to Don't make me think.
The title gives it all away, but it's very easy to forget.
You should always put an emphasis on simplicity and clarity in web design. Don't do the fireworks.

**Comment by AgathaJarynski** [comment]
One of the books that I would highly recommend is "JavaScript: The Good Parts" by Douglas Crockford. This book is a must-read for anyone who wants to become proficient in JavaScript and write clean, maintainable code.

**Comment by JohnGregorySpook** [comment]
I would recommend "The Design of Everyday Things" - it doesn't focus on software UX, but helps you look at things from a user-centered perspective.
